woocommerce/woocommerce-gutenberg-products-block
# Clean up docs

In #6405, we adjusted the documentation of this repo. Due to a merge conflict, a few mistakes slipped in. This PR applies the following changes:

- Link the translation docs to the main README.md file.
- Move docs for internal developers into the corresponding subfolder.
- Delete obsolete files.

### Testing

1. Ensure that the translation docs are linked correctly.
2. Ensure that every file, that had been removed from the main docs folder, is available in a subfolder.
